.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: actions/setup-python@v4
```

Comment:
```suggestion
    - uses: actions/setup-python@v4
      with:
        python-version: "3.x"
```

Reply:
Had to add this due to change discussed in https://github.com/actions/setup-python/issues/421. :(